Detectors/ITSMFT/ITS/workflow/src/ThresholdCalibratorSpec.cxx

```diff
@@ -393,56 +406,60 @@ void ITSCalibrator::extractThresholdRow(const short int& chipID, const short int
     catch (int i) {
       if (this->mVerboseOutput) {
         LOG(warning) << "Start-finding unsuccessful for chipID " << chipID
-                     << " row " << row << " column " << (col_i + 1) << '\n';
+                     << " row " << row << " column " << (col_i) << '\n';
       }
       continue;
     } catch (int* i) {
       if (this->mVerboseOutput) {
         LOG(warning) << "Start-finding unsuccessful for chipID " << chipID
-                     << " row " << row << " column " << (col_i + 1) << '\n';
+                     << " row " << row << " column " << (col_i) << '\n';
```

Comment:
Your findThreshold never throws an exception, what is the purpose of these catches. Also, you have 2 catches with the same action but you never use the catch argument. Usually, if one wants to catch `any` exception, one does 
`catch(...) { /* your processing code */ }  `

Reply:
you are right. Actually now that I check in detail, we print already a warning in case success = false in each of the fit methods. So I removed try try-catch block that actually was doing nothing (as you said) and in case success = false we keep the info into the c-arrays and hence in the root tree filled in ```saveThreshold```